src/test/java/apoc/load/relative/LoadXmlRelativePathTest.java

```diff
@@ -0,0 +1,268 @@
+                            "| \"bk106\" | \"Randall, Cynthia\"     | \"Lover Birds\"                            |\n" +
+                            "| \"bk107\" | \"Thurman, Paula\"       | \"Splish Splash\"                          |\n" +
+                            "| \"bk108\" | \"Knorr, Stefan\"        | \"Creepy Crawlies\"                        |\n" +
+                            "| \"bk109\" | \"Kress, Peter\"         | \"Paradox Lost\"                           |\n" +
+                            "| \"bk110\" | \"O'Brien, Tim\"         | \"Microsoft .NET: The Programming Bible\"  |\n" +
+                            "| \"bk111\" | \"O'Brien, Tim\"         | \"MSXML3: A Comprehensive Guide\"          |\n" +
+                            "| \"bk112\" | \"Galos, Mike\"          | \"Visual Studio 7: A Comprehensive Guide\" |\n" +
+                            "+-----------------------------------------------------------------------------+\n" +
+                            "12 rows\n", result.resultAsString());
+                });
+    }
+
+    @Test
+    public void testLoadXmlXpathAuthorFromBookId () {
+        testCall(db, "CALL apoc.load.xml('books.xml', '/catalog/book[@id=\"bk102\"]/author') yield value as result",
+                (r) -> {
+                    assertEquals("author", ((Map) r.get("result")).get("_type"));
+                    assertEquals("Ralls, Kim", ((Map) r.get("result")).get("_text"));
+                });
+    }
+
+    @Test
+    public void testLoadXmlXpathGenreFromBookTitle () {
+        testCall(db, "CALL apoc.load.xml('books.xml', '/catalog/book[title=\"Maeve Ascendant\"]/genre') yield value as result",
+                (r) -> {
+                    assertEquals("genre", ((Map) r.get("result")).get("_type"));
+                    assertEquals("Fantasy", ((Map) r.get("result")).get("_text"));
+                });
+    }
+
+    @Test
+    public void testLoadXmlXpathReturnBookFromBookTitle () {
+        testCall(db, "CALL apoc.load.xml('books.xml', '/catalog/book[title=\"Maeve Ascendant\"]/.') yield value as result",
+                (r) -> {
+                    Object value = r.values();
+                    assertEquals(XML_XPATH_AS_NESTED_MAP, value.toString());
+                });
+    }
+
+    @Test
+    public void testLoadXmlXpathBooKsFromGenre () {
+            testResult(db, "CALL apoc.load.xml('books.xml', '/catalog/book[genre=\"Computer\"]') yield value as result",
+                (r) -> {
+                    Map<String, Object> next = r.next();
+                    Object result = next.get("result");
+                    Map resultMap = (Map) next.get("result");
+                    Object children = resultMap.get("_children");
+
+                    List<Object>  childrenList = (List<Object>) children;
+                    assertEquals("bk101", ((Map) result).get("id"));
+                    assertEquals("author", ((Map) childrenList.get(0)).get("_type"));
```

Comment:
```suggestion
                    Map<String, String> children = ((Map<String, String>) childrenList.get(0));
                    assertEquals("author", .get("_type"));
```
e aggiustare a cascata gli altri